package.json

```diff
@@ -9,7 +9,7 @@
     "build:watch": "gulp link watch",
     "clean": "gulp clean && gulp link --clean",
     "foundry": "node foundry/resources/app/main.js --noupnp --dataPath=./data",
-    "update": "npm install --save-dev gitlab:spice_king/foundry-pc-types"
+    "update": "npm install --save-dev gitlab:a.drone.enthusiast/foundry-pc-types"
```

Comment:
Don't change deps for no good reason in a random commit.

Reply:
Alright, this won't compile against your typescript definitions without some API changes. Please accept my merge request on gitlab [here](https://gitlab.com/Spice_King/foundry-pc-types/-/merge_requests).